README.md

```diff
@@ -9,8 +9,8 @@ repo, run `make setup` and follow the instructions, then just run `make` to brin
 ## Requirements
 You must have the following tools available in order to successfully use the DM Runner:
 * Python 3 (including headers if appropriate) installed globally with `pip` and `virtualenv` packages.
-* Node v6.12.2 (consider using Node Version Manager), NPM 3+, and Yarn 1.3.2+ installed and available in your path.
-  * The command `nvm alias default 6.12.2` will alias the right version of Node for you
+* Node v8.12.0 (consider using Node Version Manager), NPM 6+, and Yarn 1.13.0+ installed and available in your path.
+  * The command `nvm use` will install and select the correct version of node for you
```

Comment:
👍

Reply:
This doesn't work for me?

Reply:
<img width="487" alt="screen shot 2019-01-24 at 10 42 22" src="https://user-images.githubusercontent.com/2920760/51672974-bf986f00-1fc4-11e9-82b5-13f95bc47c34.png">

Doesn't seem to install it for you?

Reply:
Thanks for testing, I have updated the suggested command to `nvm install && nvm use`